core/dbt/include/global_project/macros/materializations/models/materialized_view/create_or_replace_materialized_view.sql

```diff
@@ -0,0 +1,11 @@
+{% macro create_or_replace_materialized_view() %}
```

Comment:
I don't think we need both `create_as` and `create_or_replace_as`. I think we call them both `create_as`, and then use "create or replace" if it's supported. This would impact the materialization itself (whether we need to drop first or not).